ui/perfherder/alerts/AlertsView.jsx

```diff
@@ -253,6 +253,47 @@ class AlertsView extends React.Component {
               make changes
             </Alert>
           )}
+          {pageNums.length > 1 && (
```

Comment:
This whole block of code should be extracted to a component since it's being used in two places. Anytime you find yourself copying/pasting the same code, that's an indication it should probably be moved to a function/component. :)